backend/tests/test_beers_router.py

```diff
@@ -0,0 +1,345 @@
+from fastapi import FastAPI
+from fastapi.testclient import TestClient
+from peewee import *
+
+import pytest
+import random
+
+from backend.routers import beers
+from backend.models import Beer
+
+from backend.tests.utils.payload_to_string import payload_to_string
+from decimal import Decimal
+
+app = FastAPI()
+app.include_router(beers.router)
+
+db = PostgresqlDatabase(
+    "test_beerpy",
+    user="test_user",
+    password="iLoveTesting",
+    host="localhost",
+    port=5432,
+)
+
+client = TestClient(app)
+
+
+def setup_module():
+    db.bind([Beer])

```

Comment:
why not on all models?

Reply:
Why would I need it if no other model is relevant to these test?
Of course I can add them, but why?